packages/test/test-service-load/src/leaderElection.ts

```diff
@@ -9,12 +9,14 @@ import { ChildLogger, TelemetryLogger } from "@fluidframework/telemetry-utils";
 
 export class LeaderElection {
     private readonly beatInEveryNSecs: number = 1000; // 1 secs
-    private readonly leaderWait: number = 5000; // 5 secs
+    private readonly leaderWait: number = 30000; // 30 secs
     private lastPinged: number | undefined;
     private readonly logger: TelemetryLogger;
+    private prevPing: number;
 
     constructor(private readonly dataStoreRuntime: IFluidDataStoreRuntime) {
         this.logger = ChildLogger.create(this.dataStoreRuntime.logger, "SignalLeaderElection");
+        this.prevPing = 0;
```

Comment:
i don't think this should be initialize to 0. i'd rather it be undefined. 0 will lead to weird results for LeaderFound